### ISSUE TITLE
Tertiary button is missing focus styles

### ISSUE THREAD

**Issue by tw15egan** [carbon-design-system/carbon#3592]
Focus styles are added but blend in with the border and white background

**Comment by tw15egan** [comment]
Tagging @carbon-design-system/design to see what focus styles should be

**Comment by aagonzales** [comment]
This is the spec for tertiary buttons:

![image](https://user-images.githubusercontent.com/11670886/62160307-3fd3c000-b2d9-11e9-98bb-c6b212e71001.png)
![image](https://user-images.githubusercontent.com/11670886/62160331-4f530900-b2d9-11e9-82b6-2691a972c8bd.png)

Focus style:
```
Background color: $interactive-03
Text size: $body-short-01 
Text color: $text-04
Border: 2px inner stroke ($focus)
```
